backend/config/auth.config.js

```diff
@@ -1,8 +1,7 @@
 /*eslint-disable */
 module.exports = {
-  SECRET:
-    'c0d7d0716e4cecffe9dcc77ff90476d98f5aace08ea40f5516bd982b06401021191f0f24cd6759f7d8ca41b64f68d0b3ad19417453bddfd1dbe8fcb197245079',
+  SECRET: process.env.JWT_SECRET,
```

Comment:
How would you feel about pulling out this secret fix into another PR and getting the token expiration fix in now?

Reply:
Alright, dug into the actions some more. We do need to set this value for production before we merge, but to get your unit tests to pass (and to be in-line with a check we have for env values), do the following:

- Add  `envkey_JWT_SECRET ` to .github/workflows/all-PRs.yaml beneath line 47 with a dummy secret (we'll need to reference the github action secret once we add it)
- Add JWT_SECRET to the `require('assert-env')([...])` array in backend/src/app.js

Reply:
JWT_SECRET has been added to the github secrets in the dev environment.